extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/promptBeginTracking.js

```diff
@@ -26,24 +27,24 @@ function promptBeginTracking({ servicesManager, extensionManager }, ctx, evt) {
 
 function _askTrackMeasurements(uiViewportDialogService, viewportId) {
   return new Promise(function (resolve, reject) {
-    const message = 'Track measurements for this series?';
+    const message = i18n.t('MeasurementTable:Track measurements for this series?');
     const actions = [
       {
         id: 'prompt-begin-tracking-cancel',
         type: ButtonEnums.type.secondary,
-        text: 'No',
+        text: i18n.t('MeasurementTable:No'),
```

Comment:
Things like Yes and No should be in Common.json because they are applicable elsewhere.